pkgs/development/libraries/glib/default.nix

```diff
@@ -66,10 +66,9 @@ stdenv.mkDerivation rec {
     ++ optionals stdenv.isLinux [ utillinuxMinimal ] # for libmount
     ++ optionals doCheck [ tzdata libxml2 desktop-file-utils shared-mime-info ];
 
-  nativeBuildInputs = [ pkgconfig gettext perl python ];
+  nativeBuildInputs = [ pkgconfig gettext perl python libintl libiconv ];
 
-  propagatedBuildInputs = [ zlib libffi libiconv ]
-    ++ libintlOrEmpty;
+  propagatedBuildInputs = [ zlib libffi ];
```

Comment:
I'm not sure if it's right or wrong, but noticed this (while investigating problems w/musl and missing libiconv)--
what's the rationale for removing libiconv from propagated build inputs here?

Reply:
We can have it propagate if thia breaks things. The rationale is that things that depend on glib shouldnt need to know about libiconv (esp. setup hooks). Not sure if it breaks glib though?